src/components/MDXContainer.js

```diff
@@ -4,21 +4,29 @@ import cx from 'classnames';
 import { MDXRenderer } from 'gatsby-plugin-mdx';
 import { MDXProvider } from '@mdx-js/react';
 
-import Step from './Step';
-import Steps from './Steps';
 import Caution from './Caution';
 import Important from './Important';
 import Tip from './Tip';
 import Intro from './Intro';
 import Iframe from './Iframe';
+import Tutorial from './Tutorial';
+import TutorialStep from './TutorialStep';
+import TutorialSection from './TutorialSection';
+import Project from './Project';
 import { MDXCodeBlock, Video } from '@newrelic/gatsby-theme-newrelic';
 
 import styles from './MDXContainer.module.scss';
 
 const components = {
   Video,
-  Step,
-  Steps,
+  // Remove these when all step/steps components have been updated to the new
+  // name
+  Step: TutorialStep,
+  Steps: TutorialSection,
+  Project,
+  Tutorial,
+  TutorialStep,
+  TutorialSection,
```

Comment:
I'm not a huge fan of the new names. Would love to get some help on naming these components. We've received feedback that `Steps` and `Step` are too similar and hard to follow. The old names are aliased above, but I'd love to figure out a better name for this component.